src/mpid/ch4/generic/am/mpidig_am_recv_utils.h

```diff
@@ -287,6 +325,11 @@ MPL_STATIC_INLINE_PREFIX bool MPIDIG_recv_initialized(MPIR_Request * rreq)
     return MPIDIG_REQUEST(rreq, req->recv_async).recv_type != MPIDIG_RECV_NONE;
 }
 
+MPL_STATIC_INLINE_PREFIX MPI_Aint MPIDIG_recv_in_data_sz(MPIR_Request * rreq)
```

Comment:
Where is this used?

Reply:
In the transport rndv callback, it needs to know how much data was promised to come.